Ginger/GingerCoreCommon/UIElement/ElementProperty.cs

```diff
@@ -0,0 +1,38 @@
+﻿#region License
+/*
+Copyright © 2014-2020 European Support Limited
+
+Licensed under the Apache License, Version 2.0 (the "License")
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at 
+
+http://www.apache.org/licenses/LICENSE-2.0 
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS, 
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. 
+See the License for the specific language governing permissions and 
+limitations under the License. 
+*/
+#endregion
+
+namespace Amdocs.Ginger.Common.UIElement
+{
+    public static class ElementProperty
+    {
+        public const string IsPOMWidgetElement = "IsWidgetElement";
+        public const string ParentBrowserPath = "ParentBrowserPath";
+        public const string ElementType = "Element Type";
+        public const string PlatformElementType = "Platform Element Type";
+        public const string ParentIFrame = "Parent IFrame";
+        public const string XPath = "XPath";
+        public const string RelativeXPath = "Relative XPath";
+
+        public const string Height = "Height";
```

Comment:
Codacy found an issue: [Change this constant to a 'static' read-only property.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5877667)